cleverhans/attacks_tf.py

```diff
@@ -57,6 +63,38 @@ def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
     return adv_x
 
 
+def fgsm_iterative(x, y, model, eps, eps_iter, n_iter,
+                   clip_min=None, clip_max=None):
+    """
+    TensorFlow implementation of the iterative Fast Gradient
+    Sign method.
+    :param x: the input placeholder
+    :param y: the output placeholder
+    :param model: the Keras model that we want to attack
+    :param eps: the maximum input variation parameter
+    :param eps_iter: the input variation for each iteration
+    :param n_iter: the number of iterations to run
+    :param clip_min: optional parameter that can be used to set a minimum
+                    value for components of the example returned
+    :param clip_max: optional parameter that can be used to set a maximum
+                    value for components of the example returned
+    :return: a tensor for the adversarial example
```

Comment:
Maybe put a link to the paper in the docstring so users can read about what it is